client/lib/translator-jumpstart/index.js

```diff
@@ -84,7 +84,15 @@ communityTranslatorJumpstart = {
 			optionsFromPage = {};
 		}
 
-		props = { className: 'translatable' };
+		if ( 'string' !== typeof originalFromPage ) {
+			debug( 'unknown original format' );
+			return displayedTranslationFromPage;
+		}
+
+		const props = {
+			className: 'translatable',
+			'data-singular': originalFromPage
```

Comment:
There was no change in functionality, but the diff shown here was missing something from the current version. See here: https://github.com/Automattic/wp-calypso/blob/master/client/lib/translator-jumpstart/index.js#L89

But there was some duplicate code that I've now removed.